shoop/core/models/_shipments.py

```diff
@@ -43,7 +43,7 @@ class Shipment(models.Model):
     tracking_code = models.CharField(max_length=64, blank=True, verbose_name=_("tracking code"))
     description = models.CharField(max_length=255, blank=True, verbose_name=_("description"))
     volume = MeasurementField(unit="m3", verbose_name=_("volume"))
-    weight = MeasurementField(unit="kg", verbose_name=_("weight"))
+    weight = MeasurementField(unit="g", verbose_name=_("weight"))
```

Comment:
I think the weight should be in kg as it used to be, but whoever is calculating that weight should do the conversion from grams to kilograms. I.e. the bug is not here, but in the calculation side..